components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -360,102 +375,27 @@ Rules of type `Node` can have conditions with attributes `Chef Organization`, `C
 Navigate to the project list page once more. Select `Update Projects`. Upon completion of the update, you should be able to
 filter by `project-devops` across Automate's dashboards and see only the ingested data that you expect.
 
-#### Project Owners
-
-The role Project Owner allows admin users to delegate management of project membership to another user without granting that user access to other admin actions in Chef Automate. Project Owners have the following permissions on resources assigned to their project:
-
-```text
-"infra:*",
-"compliance:*",
-"system:*",
-"event:*",
-"ingest:*",
-"secrets:*",
-"telemetry:*",
-"iam:projects:list",
-"iam:projects:get",
-"iam:projects:assign",
-"iam:policies:list",
-"iam:policies:get",
-"iam:policyMembers:*",
-"iam:teams:list",
-"iam:teams:get",
-"iam:teamUsers:*",
-"iam:users:get",
-"iam:users:list"
-```
-
-In order to create a Project Owner, you can use an existing user, or create a user with username `test_project_owner` by following [Creating Users]({{< relref "users.md#manage-local-users-from-the-ui" >}}).
-
-Next, you will create a policy that gives `test_project_owner` the role of `project-owner` for your new project. The policy will look like the JSON below:
-
-```json
-{
-  "name": "Project Devops Policy",
-  "id": "project-devops-policy",
-  "projects": ["project-devops"],
-  "members": [ "user:local:test_project_owner"],
-  "statements": [
-    {
-      "effect": "ALLOW",
-      "role": "project-owner",
-      "projects": ["project-devops"]
-    }
-  ]
-}
-```
-
-By adding `project-devops` to the policy's top-level `projects` field, we ensure that the Project Owner has access to this policy.
-
-Save as a JSON file and follow the steps in [Creating a Policy]({{< relref "iam-v2-api-reference.md#creating-a-policy" >}}) to send that data to Chef Automate.
-
-#### Editor and Viewer Policies for Projects
+#### Project Policies
```

Comment:
With the changes in this PR to auto-generate the three support policies, the bulk of the next couple sections is no longer needed, as it was describing how to manually create those same policies.

Reply:
🎉

Reply:
(although are we just gonna want to add them back once we make generating the policies optional? 🤔

Reply:
Good question, @blakestier 	!